### DISCUSSION TITLE
How to calculate the GMFE?

### DISCUSSION THREAD

**Discussion by andanteyang** [Open-Systems-Pharmacology/Forum#839] [Questions & Problems] [answered]
Hi everyone,

Is there a function for calculating the GMFE and/or plotting the GOFMergedPlots embedded in PKsim/Mobi? If there is any other parameters to evaluate/compare the fitness between different models? 

Many thanks!

**Comment by prvmalik** [comment, marked as answer]
At present, there is no automated way to do this. I use 2 manual methods, depending on the amount of data that I am working with.

A. Export the simulation results to .csv and manually calculate predicted/observed.

B. Run a parameter identification with all the desired simulations for evaluation. Set 0 identification runs, linear scaling on the datasets and linear scaling on the parameters (very important!). Then you can export the time profile analysis and the residual analysis to get the data that you need for GMFE on a larger scale.

When doing this, you must be extremely careful that the units of the concentration are all the same for the observed datasets and the simulations.
